mono/tests/Makefile.am

```diff
@@ -632,7 +673,11 @@ AOT_DISABLED_TESTS=constraints-load.exe
 TEST_CSC_SRC=			\
 	vararg.cs
 
-TEST_IL_SRC=			\
+IL_SRC_MOBILE_STATIC_NOT_SUPPORTED=	\
+	constraints-load.il \
```

Comment:
On AOT, we will try to resolve the type BrokenIComparable from mscorlib at AOT time, which will fail. This test is disabled because it cannot be AOTed.